Add \choice and \object-list fields in IDD for Inlet and Supply Side Mixers

wfbuhl opened the pull request:
AirTerminal:SingleDuct:InletSideMixer and AirTerminal:SingleDuct:OutletSideMixer had very minimal IDD descriptions. Adding \choice and \object-list fields define the allowed input: only 4 pipe fan coils and water to air heat pumps can be used as part of these terminal units. Addresses #4530.

wfbuhl committed:
In AirTerminal:SingleDuct:InletSideMixer and AirTerminal:SingleDuct:SupplySideMixer make "\field ZoneHVAC Terminal Unit Object Type" a choice field with choices ZoneHVAC:FourPipeFanCoil and ZoneHVAC:WaterToAirHeatPump. Use \object-list in "\field ZoneHVAC Terminal Unit Name".

Fix IO Ref for AirTerminal:SingleDuct:InletSideMixer and AirTerminal:SingleDuct:SupplySideMixer.

wfbuhl commented:
Updated title and first comment. This is ready to merge into the release branch.

mjwitte committed:
Additional IDD updates for airterminal mixers

mjwitte committed:
Merge remote-tracking branch 'remotes/origin/develop' into Issue4530UpdateIDDandIORefDOAToZonalUnits

mjwitte commented:
@wfbuhl I added some \node tags and expanded the \memos.  Also, noticed that AirTerminal:SingleDuct:SupplySideMixer, Terminal Unit Secondary Air Inlet Node Name wasn't marked as a required-field.  I added that.  Let me know if it should be an optional field.

wfbuhl commented:
This looks OK to me

On Mon, Sep 14, 2015 at 11:30 AM, Michael J. Witte <notifications@github.com

> wrote:
> 
> @wfbuhl https://github.com/wfbuhl I added some \node tags and expanded
> the \memos. Also, noticed that AirTerminal:SingleDuct:SupplySideMixer,
> Terminal Unit Secondary Air Inlet Node Name wasn't marked as a
> required-field. I added that. Let me know if it should be an optional field.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/NREL/EnergyPlus/pull/5180#issuecomment-140167900.
